{"data_type": "CVE", "data_format": "MITRE", "data_version": "4.0", "CVE_data_meta": {"ID": "CVE-2018-14651", "ASSIGNER": "cve@mitre.org"}, "affects": {"vendor": {"vendor_data": []}}, "problemtype": {"problemtype_data": [{"description": []}]}, "references": {"reference_data": [{"url": "https://access.redhat.com/errata/RHSA-2018:3431", "name": "RHSA-2018:3431", "refsource": "REDHAT", "tags": []}, {"url": "https://access.redhat.com/errata/RHSA-2018:3432", "name": "RHSA-2018:3432", "refsource": "REDHAT", "tags": []}, {"url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-14651", "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-14651", "refsource": "CONFIRM", "tags": []}, {"url": "https://lists.debian.org/debian-lts-announce/2018/11/msg00003.html", "name": "[debian-lts-announce] 20181105 [SECURITY] [DLA 1565-1] glusterfs security update", "refsource": "MLIST", "tags": []}]}, "description": {"description_data": [{"lang": "en", "value": "It was found that the fix for CVE-2018-10927, CVE-2018-10928, CVE-2018-10929, CVE-2018-10930, and CVE-2018-10926 was incomplete. A remote, authenticated attacker could use one of these flaws to execute arbitrary code, create arbitrary files, or cause denial of service on glusterfs server nodes via symlinks to relative paths."}]}}
